test/integration/node-specific/client_close.test.ts

```diff
@@ -430,7 +427,7 @@ describe.skip('MongoClient.close() Integration', () => {
         const metadata: MongoDBMetadataUI = { requires: { topology: 'sharded' } };
 
         describe('after SRVPoller is created', () => {
-          it('timers are cleaned up by client.close()', metadata, async () => {
+          it.skip('timers are cleaned up by client.close()', metadata, async () => {
             const run = async function ({ MongoClient, expect, getTimerCount }) {
               const SRV_CONNECTION_STRING = `mongodb+srv://test1.test.build.10gen.cc`;
```

Comment:
`TypeError: Cannot read properties of undefined (reading 'libuvResources')`

something's up with this process' exit

Reply:
Might help to attach a message to whatever is printing "undefined" 🤔

Reply:
Does it work for you now? I lowered the serverSelectionTimeoutMS and[ it's not timing out but rather just printing the correct resource clean-up error now](https://parsley.mongodb.com/test/mongo_node_driver_next_rhel80_large_hydrogen_test_6.0_sharded_cluster_patch_fd902d31d5cb65100f1c8b6f8811886b44ba19d5_6793f363e851a1000722e46f_25_01_24_20_10_36/0/6540287b526ab531ffc067a261c9a486?bookmarks=0,10&shareLine=0).